scipy/signal/_signaltools.py

```diff
@@ -3109,7 +3109,10 @@ def resample(x, num, t=None, axis=0, window=None, domain='time'):
 
     As noted, `resample` uses FFT transformations, which can be very
     slow if the number of input or output samples is large and prime;
-    see `scipy.fft.fft`.
+    see `scipy.fft.fft`. In such cases, it can be faster to use the polyphase
```

Comment:
A little cosmetics: Writing  ```:func:`~scipy.fft.fft` ``` instead of  `` `scipy.fft.fft` `` produces the shortened link [fft](https://scipy.github.io/devdocs/reference/generated/scipy.fft.fft.html) instead of [scipy.fft.fft](https://scipy.github.io/devdocs/reference/generated/scipy.fft.fft.html). Same with `` `scipy.signal.resample_poly` ``. It's matter of taste, I guess.